### ISSUE TITLE
Improve web

### ISSUE THREAD

**Issue by angrykoala** [angrykoala/gaucho#64]
The web (https://angrykoala.github.io/gaucho/) could be improved.

Add more stuff, improve styles, fix license link etc...

**Comment by dmaroulidis** [comment]
Hello, what do you mean with fix license link?

**Comment by angrykoala** [comment]
There was an issue about the link to gaucho license, as you can see it was already fixed in ac80c2239e03a584ede15e2ae463078773dc2bf7

**Comment by angrykoala** [comment]
This issue is a general update and overall improvements of the webpage https://angrykoala.github.io/gaucho/

**Comment by JesseSingleton** [comment]
I potentially could help with this but have you got a better list of requirements/tasks? 'Add more stuff, improve styles' is very vague. I think the overall design is fine but not sure what you want improved. Do you still want a single landing page? Multiple pages? More docs etc.

**Comment by angrykoala** [comment]
Well, my level when it comes to web design is null, so basically the requirements are vague because I don't know what could be better, but a list of my ideas (which, can, and should, be discussed):
* Update with some screenshots of the app (the screenshots are already uploaded to gh-pages)
* Update the text (its outdated since I wrote it, check the difference with `dev` readme)
* Update materialize library and general improvement in design (if it makes sense)

Personally, I'm ok with the single landing page, unless anyone has a better idea. The main problem is probably the actual text, and having too much text, the idea of the page is to be a first overview of the app (this thing is cool and whatever...), while currently is just an old readme.

So, basically any improvement on making the web more appealing to people who don't know what Gaucho is would be welcome

**Comment by angrykoala** [comment]
Done as part of 0.5 milestone
